dev/tools/gen_defaults/bin/gen_defaults.dart

```diff
@@ -104,17 +105,18 @@ Future<void> main(List<String> args) async {
   tokens['colorsDark'] = _readTokenFile('color_dark.json');
 
   AppBarTemplate('AppBar', '$materialLib/app_bar.dart', tokens).updateFile();
+  BannerTemplate('Banner', '$materialLib/banner.dart', tokens).updateFile();
   ButtonTemplate('md.comp.elevated-button', 'ElevatedButton', '$materialLib/elevated_button.dart', tokens).updateFile();
   ButtonTemplate('md.comp.filled-button', 'FilledButton', '$materialLib/filled_button.dart', tokens).updateFile();
   ButtonTemplate('md.comp.filled-tonal-button', 'FilledTonalButton', '$materialLib/filled_button.dart', tokens).updateFile();
   ButtonTemplate('md.comp.outlined-button', 'OutlinedButton', '$materialLib/outlined_button.dart', tokens).updateFile();
   ButtonTemplate('md.comp.text-button', 'TextButton', '$materialLib/text_button.dart', tokens).updateFile();
   CardTemplate('Card', '$materialLib/card.dart', tokens).updateFile();
   CheckboxTemplate('Checkbox', '$materialLib/checkbox.dart', tokens).updateFile();
-  ChipActionTemplate('ActionChip', '$materialLib/chip_action.dart', tokens).updateFile();
-  ChipFilterTemplate('FilterChip', '$materialLib/chip_filter.dart', tokens).updateFile();
-  ChipFilterTemplate('FilterChip', '$materialLib/chip_choice.dart', tokens).updateFile();
-  ChipInputTemplate('InputChip', '$materialLib/chip_input.dart', tokens).updateFile();
+  ChipActionTemplate('ActionChip', '$materialLib/action_chip.dart', tokens).updateFile();
+  ChipFilterTemplate('FilterChip', '$materialLib/filter_chip.dart', tokens).updateFile();
+  ChipFilterTemplate('FilterChip', '$materialLib/choice_chip.dart', tokens).updateFile();
+  ChipInputTemplate('InputChip', '$materialLib/input_chip.dart', tokens).updateFile();
```

Comment:
rename template class names to `*Chip`, as per our discussion over my PR https://github.com/flutter/flutter/pull/110910.

Reply:
since there are more than just a few changes, i felt it deserves a separate PR. i've raised it #111364, addressing them and adding a test so as to prevent it in the future. hope that's okay. sorry for the back and forth.

Reply:
That's good, we can reland this after your PR, in case of any conflict.